addons/website_sale/models/res_company.py

```diff
@@ -15,3 +15,15 @@ def action_open_website_sale_onboarding_payment_acquirer(self):
         self.env.company.payment_onboarding_payment_method = 'stripe'
         menu_id = self.env.ref('website.menu_website_dashboard').id
         return self._run_payment_onboarding_step(menu_id)
+
+    def _get_default_pricelist_vals(self, values=None):
+        """Add values to the default pricelist at company creation or activation of the pricelist
```

Comment:
Maybe mention why the `website_id` is set to `False` and the use of the override instead of c/p the docstring of the base method?